kategory-test/src/main/kotlin/kategory/laws/FunctorFilterLaws.kt

```diff
@@ -0,0 +1,31 @@
+package kategory.laws
+
+import io.kotlintest.properties.Gen
+import io.kotlintest.properties.forAll
+import kategory.*
+
+object FunctorFilterLaws {
+
+    inline fun <reified F> laws(AP: Applicative<F> = applicative(), FFF: FunctorFilter<F> = functorFilter(), EQ: Eq<HK<F, Int>>): List<Law> =
```

Comment:
Some FunctorFilters may not be Applicatives. I'd suggest passing a constructor, and inlining AP::pure in the current callsites instead.